oncotator/output/VcfOutputRenderer.py

```diff
@@ -184,35 +183,31 @@ def _renderSortedTsv(self, templateFilename, vcfFilename, tsvFilename, sampleNam
 
         ctr = 0
         m = None
-        try:
-            for m in tsvReader:
-                ctr += 1
-                isNewRecord = self._isNewVcfRecordNeeded(chrom, m["chr"], pos, m["start"], refAllele, m["ref_allele"])
-                if isNewRecord:
-                    if recordBuilder is not None:
-                        record = recordBuilder.createRecord()
-                        vcfWriter.write_record(record)
-                        index += 1
-                        if index % nrecords == 0:
-                            self.logger.info("Rendered " + str(index) + " vcf records.")
-                            vcfWriter.flush()
-
-                    chrom = m["chr"]
-                    pos = m["start"]
-                    refAllele = m["ref_allele"]
-
-                    recordBuilder = RecordBuilder(chrom, int(pos), refAllele, sampleNames)
-
-                recordBuilder = self._parseRecordBuilder(m, recordBuilder, dataManager, inferGenotypes)
-
-            if recordBuilder is not None:
-                record = recordBuilder.createRecord()
-                vcfWriter.write_record(record)
-            vcfWriter.close()
-
-        except Exception as e:
-            self.logger.error(traceback.format_exc())
-            self.logger.error("Error at mutation " + str(ctr) + " " + str([m["chr"], m["start"], m["end"]]) + ": ")
+
+        for m in tsvReader:
+            ctr += 1
+            isNewRecord = self._isNewVcfRecordNeeded(chrom, m["chr"], pos, m["start"], refAllele, m["ref_allele"])
+            if isNewRecord:
+                if recordBuilder is not None:
+                    record = recordBuilder.createRecord()
+                    vcfWriter.write_record(record)
+                    index += 1
+                    if index % nrecords == 0:
+                        self.logger.info("Rendered " + str(index) + " vcf records.")
+                        vcfWriter.flush()
+
+                chrom = m["chr"]
+                pos = m["start"]
+                refAllele = m["ref_allele"]
+
+                recordBuilder = RecordBuilder(chrom, int(pos), refAllele, sampleNames)
+
+            recordBuilder = self._parseRecordBuilder(m, recordBuilder, dataManager, inferGenotypes)
+
+        if recordBuilder is not None:
+            record = recordBuilder.createRecord()
+            vcfWriter.write_record(record)
+        vcfWriter.close()
```

Comment:
Doesn't tmpvcfReader need to be closed as well?  They could both be moved into a `with` block potentially?